Features/single service page

AbudZohair opened the pull request:
- the builtin bootstrap carousel used
- if the users entered the id param manually, it would lead to an error (how can I handle this?)
- the JSON file data will be static? I don't fetch any data

AbudZohair committed:
Add singleServicePage Component

AbudZohair committed:
Add singleServicePage styling

AbudZohair committed:
specify the active calss to navBar item

- specify the global active class to navBar item
- carousel coflict while using rhe active class resolved

dvmhmdsd commented on src/modules/teamServices/components/singleServicePage/index.js:
```diff
@@ -1,10 +1,92 @@
+  constructor(props) {
+    super(props);
+    const { id } = this.props.match.params;
+    this.state.service = services.filter((service) => service.id === id)[0];
```
1. Use `setState()`.

2. Use `.find()` instead of `.filter()`.

dvmhmdsd reviewed (changes requested):
Add support for `focus` of carousel indicator to improve accessibility.

dvmhmdsd commented on src/modules/teamServices/components/singleServicePage/index.js:
```diff
@@ -1,10 +1,92 @@
+        </Helmet>
+        <Layout>
+          <div className="container">
+            <header className={`${styles["single-service__header"]}`}>
```
Gallery will not always be found, some services will not have gallery.

dvmhmdsd reviewed (changes requested):
Solve the errors in the console.

AbudZohair committed:
Replace filter method by find and  use setState

AbudZohair committed:
Add Enhancements to the layout.

- Add service Cover
- Add gallery conditional rendaring
- add focus on carousel indicators (not accessible via Enter)

AbudZohair committed:
Add styling for layout Enhancements

AbudZohair committed:
Improve carousel controls UX
